src/main/java/com/docker/rihal/RihalApplication.java

```diff
@@ -1,13 +1,21 @@
 package com.docker.rihal;
 
+import io.github.cdimascio.dotenv.Dotenv;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class RihalApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(RihalApplication.class, args);
-	}
+    public static void main(String[] args) {
+        // Load environment variables from .env file
+        Dotenv dotenv = Dotenv.load();
+        System.setProperty("SPRING_DATASOURCE_URL", dotenv.get("SPRING_DATASOURCE_URL"));
+        System.setProperty("SPRING_DATASOURCE_USERNAME", dotenv.get("SPRING_DATASOURCE_USERNAME"));
+        System.setProperty("SPRING_DATASOURCE_PASSWORD", dotenv.get("SPRING_DATASOURCE_PASSWORD"));
+        System.setProperty("SPRING_JPA_HIBERNATE_DDL_AUTO", dotenv.get("SPRING_JPA_HIBERNATE_DDL_AUTO"));
+        System.setProperty("SPRING_JPA_SHOW_SQL", dotenv.get("SPRING_JPA_SHOW_SQL"));
```

Comment:
Is this really needed ? 
application.properties are reading the environment variables exposed by the container and setting it.
This happens when we do docker-compose up